doc/protocols/scion-header.rst

```diff
@@ -408,3 +408,425 @@ Because of its special structure, no PathMeta header is needed. There is only a
+    C_{\text{EPIC}} &= 0x391c
+    \end{align}
+
+Note that the input to the MAC function now consists of 15 instead of 
+11 bytes, but still requires only one encryption operation for block 
+cipher based MACs. Also note that :math:`\beta_{0}^EPIC = \beta_{0}`. 
+Because the MAC-calculations are different, :math:`\beta_{i}^EPIC 
+\neq \beta_{i}` (for i > 0), however.
+
+**Data plane:**
+The source host fetches the path, including all the 16-byte EPIC 
+authenticators, from the path server. It also retrieves the 
+host-level DRKeys (:math:`K_i^S` between the source host and the 
+on-path ASes, and :math:`K_{SD}` between the source host and the 
+destination host) from the certificate server. The source then 
+calculates the Hop Validation Fields (:math:`V_i`) and the 
+Destination Validation Field (:math:`V_{\text{SD}}`):
+
+.. math::    
+    \begin{align}
+    V_i &= \text{MAC}_{K_i^{\text{S}}}(\text{PacketTimestamp}, 
+        \text{Origin}, \sigma_i^{\text{EPIC}}, \text{PayloadLen})
+        ~\text{[0:4]} \\
+    V_{\text{SD}} &= \text{MAC}_{K_{\text{SD}}}
+        (\text{PacketTimestamp, Path, Payload}) \\
+    where \\
+    \text{Path} &= (\text{TsPath}, \text{Address Header}, 
+        HI_1, ..., HI_n)\\
+    HI_i &= (\text{ExpTime}_i, \text{ConsIngress}_i, 
+        \text{ConsEgress}_i) \\
+    \end{align}
+
+Depending on the implementation of the MAC, we also need to prepend 
+the length of the input (additional 2 bytes). As in SCION, the source 
+writes all the :math:`\sigma_i^{\text{EPIC}}` / 
+:math:`\sigma_i^{\text{P, EPIC}}` to the MAC-subfield of the Hop 
+Fields, but in this case truncates the MAC to 2 bytes instead of 6 
+bytes. The :math:`V_i` are subsequently stored in the HVF-subfield of 
+the Hop Fields, and :math:`V_{\text{SD}}` in the DVF field. 
+The source host writes the necessary 
+:math:`\beta` to the SegID of the Info Fields as in standard SCION.
+
+The border routers perform the same operations as in SCION (see 
+"Path Calculation" in the SCION Path Type section), but using 
+:math:`\beta_i^\text{EPIC}`, :math:`\sigma_i^\text{EPIC}` and 
+:math:`\sigma_i^\text{P, EPIC}` instead of :math:`\beta_i`, 
+:math:`\sigma_i` and :math:`\sigma_i^P`. This is possible, because 
+the Hop Fields in EPIC-SAPV still contain the two bytes of the MAC 
+that are necessary for the chaining of the hops.
+In addition, the border routers derive the necessary DRKey 
+(:math:`K_i^S`), and recompute and validate the :math:`V_i`.
+
+Upon receiving a packet, the destination host fetches :math:`K_{SD}` 
+from its local certificate server, recomputes :math:`V_{\text{SD}}` 
+and performs validation by comparing it to the DVF in the packet. 
+
+
+Details of the EPIC Path Types
+==============================
+
+Configuration
+-------------
+Network operators should be able to clearly define which kind of 
+traffic (SCION, EPIC-HP, EPIC-SAPV, and other protocols) they want to
+allow. 
+Therefore, for each AS and every interface pair, an AS can be 
```

Comment:
The order of the interface pair defines the direction, correct? That means that it is possible to allow SCION traffic from IF 1 to IF 2 but only EPIC from IF 2 to IF 1, correct?

Reply:
Yes, exactly. I added some further description to make it more clear now.